### PR TITLE
Introduce run-pylint.sh and .pylintrc

### PR DESCRIPTION
**Pull request by avrabe**
The docker container includes now pylint.
The .pylintrc should be use by pylint
and codacy.
run-pylint.sh tests all changed and untracked
python files. If a commit id is provided as first
argument also all changed python files upto the
provided commit id will be tested.

Signed-off-by: Ralf Anton Beier <ralf_beier@me.com>

### CODE REVIEW

File: run-pylint.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```
Comment (vguerra) [inline run-pylint.sh:1]: Just minor thing: some sections of this bash file are not correctly indented. Can you pls indent correctly? Just for readability.

Thanks!

Comment (avrabe) [reply run-pylint.sh:1]: Rebased and fixed the indentation.

**Comment by avrabe** [comment]
Fixes #8

**Comment by vguerra** [comment]
lgtm

**Pull request merged**